src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -782,13 +801,15 @@ IS 'inherited from dm_sysobject_r, for r_object_type of moj_media';
 CREATE TABLE media_request
 (mer_id                      INTEGER                       NOT NULL
 ,hea_id                      INTEGER                       NOT NULL
-,requestor                   INTEGER                       NOT NULL  -- FK to moj_user.moj_usr_id
-,request_status              CHARACTER VARYING
-,request_type                CHARACTER VARYING
+,requestor                   INTEGER                       NOT NULL  -- FK to user_account
+,current_owner               INTEGER                       NOT NULL  -- FK to user_account

```

Comment:
@barry-jones50410 What's this field for? I don't have it currently in my pull request for media_request table updates in flyway.  FYI @hemantasharma1129 

https://github.com/hmcts/darts-api/pull/367/files#diff-8be848af5cd5da457379f14c0bb81ab6e243dbe62ef5377f7c0ac997761a8660

Reply:
@barry-jones50410 Is the new column meant for change of ownership functionality in Admin?

Reply:
My understanding is that the change, is to allow for the media_request  where the original requestor may now differ from the person expected to receive the media requested, by recording current_owner. That is  the media when its made available, is accessible to the current_owner as opposed to the original requestor,